### ISSUE TITLE
Graphiql schema does not reload when import types using an alias

### ISSUE THREAD

**Issue by dagumak** [absinthe-graphql/absinthe#1139]
I couldn't find any documentation or reports, so I apologize if it's a known issue. I just spent an hour debugging this and found out that the schema does not reload if you use `import_types` with an alias.

For example, this works:
```elixir
  import_types(HelloWeb.Schema.Types.TestType)
```

This works when recompiling everything, but not on subsequence changes in file with that is being imported:
```elixir

  alias HelloWeb.Schema.{Types, Queries, Mutations}
  import_types(HelloWeb.Schema.Types.{
    TestType,
    HelloType
  })
```

**Comment by benwilson512** [comment]
You're saying the old schema information remains even after you reload the page?

**Comment by dagumak** [comment]
> You're saying the old schema information remains even after you reload the page?

Yeah, that is correct. I clicked the "Reload Schema" button manually and I can see it hitting the server trigger a compile as well. However, when I make changes directly in the schema file, it reloads as expected.

**Comment by dagumak** [comment]
@benwilson512 were you able to confirm this? do you need more information from me?

**Comment by benwilson512** [comment]
I have not had time to dig into this, sorry. For now I would simply avoid using the problematic aliases.

**Comment by dagumak** [comment]
> I have not had time to dig into this, sorry. For now I would simply avoid using the problematic aliases.

No rush on my side, but just wanted to see if it was reproducible by others. For now, I'm avoiding aliases and will refactor in the future.

**Comment by zvlasic** [comment]
Can confirm it happens to me also.

**Comment by zvlasic** [comment]
Let me add some info. If I use

```elixir
import_types(Project.{
   Module1,
   Module2
}
```
and do some changes on type (let's say - renaming a field), these changes would not be seen in test env, so I had to to force recompile in test env every time I did some changes.

I just found this issue and changed schema to
```elixir
import_types(Project.Module1)
import_types(Project.Module2)
```
and it solved the problem!

**Comment by kyleboe** [comment]
Confirmed here as well. I have to run a `mix clean` before starting up whenever one of the imported files changes.

**Comment by conradwt** [comment]
I haven't been able to reproduce this issue after upgrading to both Erlang 24.2 and Elixir 1.13.  Furthermore, I recently refactored the GraphQL layer that did remove all the aliases from top-level `schema.ex` file.  When changing the types of any of the `import_types`, I simply just refresh the browser and I'm all good.

**Comment by linomassarani** [comment]
Same thing here, the `Absinthe.Schema.Notation.import_types/2` macro is working fine, I've tested it for a while, it does the job.

But there is still a problem when a imported type changes, for example, if I have this on the `Schema`:

```elixir
defmodule MyWeb.API.Users.Schema do
  @moduledoc false
  use Absinthe.Schema

  alias __MODULE__
  
  import_types(Schema.{User})  # Attention to the "{"User"}"
  [...]
```

And `User` module has something like this working fine:  

```elixir
defmodule MyWeb.API.Users.Schema.User do
[...]

  object :profile do
    field(:cool_profile_field, :string, resolve: &Resolver.get_cool_profile_field/3)
  end
[...]
```

If I change to something like `field(:cool_profile_field_changed, :string, resolve: &Resolver.get_cool_profile_field_changed/3)`, it wont work anymore until I recompile everything.

The result for the `Absinthe.run/4` would be:

```elixir
  {:ok,
   %{
     errors: [
       %{
         locations: [%{column: 11, line: 9}],
         message:
           "Cannot query field \"cool_profile_field_changed\" on type \"Profile\". Did you mean \"coolProfileField\"?"
       }
     ]
   }}
```

But if you change to this all the changes will take effect:

```elixir
defmodule MyWeb.API.Users.Schema do
  @moduledoc false
  use Absinthe.Schema

  alias __MODULE__
  
  import_types(Schema.User)
  [...]
```

This odd.

Erlang/OTP 24 [erts-12.2.1]
Elixir 1.13.0
Absinthe 1.7.0

**Comment by linomassarani** [comment]
Same thing here, the `Absinthe.Schema.Notation.import_types/2` macro is working fine, I've tested it for a while, it does the job.

But there is still a problem when a imported type changes, for example, if I have this on the `Schema`:

```elixir
defmodule MyWeb.API.Users.Schema do
  @moduledoc false
  use Absinthe.Schema

  alias __MODULE__
  
  import_types(Schema.{User})  # Attention to the "{"User"}"
  [...]
```

And `User` module has something like this working fine:  

```elixir
defmodule MyWeb.API.Users.Schema.User do
[...]

  object :profile do
    field(:cool_profile_field, :string, resolve: &Resolver.get_cool_profile_field/3)
  end
[...]
```

If I change to something like `field(:cool_profile_field_changed, :string, resolve: &Resolver.get_cool_profile_field_changed/3)`, it wont work anymore until I recompile everything.

The result for the `[Absinthe.run/4](http://absinthe.run/4)` would be:

```elixir
  {:ok,
   %{
     errors: [
       %{
         locations: [%{column: 11, line: 9}],
         message:
           "Cannot query field \"cool_profile_field_changed\" on type \"Profile\". Did you mean \"coolProfileField\"?"
       }
     ]
   }}
```

But if you change to this all the changes will take effect:

```elixir
defmodule MyWeb.API.Users.Schema do
  @moduledoc false
  use Absinthe.Schema

  alias __MODULE__
  
  import_types(Schema.User)
  [...]
```

This odd.

Erlang/OTP 24 [erts-12.2.1]
Elixir 1.13.0
Absinthe 1.7.0

**Comment by jnicklas** [comment]
Can confirm as well. Took me a long while to figure out that this is related to the `import_types Quox.{Foo,Bar}` form. Importing individually fixes the problem.
